pom.xml

```diff
@@ -168,20 +156,32 @@
       </resource>
     </resources>
 
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugings</groupId>
-          <artifactId>maven-war-plugin</artifactId>
-          <version>2.6</version>
-          <configuration>
-            <warSourceDirectory>src/main/webapp</warSourceDirectory>
-            <warName>core</warName>
-            <failOnMissingWebXml>false</failOnMissingWebXml>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
+    <plugins>
+      <plugin>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>${org.eclipse.jetty.version}</version>
+        <configuration>
+          <webApp>
+            <contextPath>/core/</contextPath>
+          </webApp>
+          <httpConnector>
+            <port>8081</port>
```

Comment:
For the remote repository I was referring to the mistake I have done before. Is this something wrong with the place I had pushed to? (Please say no)